x-pack/plugins/actions/server/routes/__jest__/_mock_server.ts

```diff
@@ -54,13 +54,13 @@ export function createMockServer(config: Record<string, any> = defaultConfig) {
   server.register({
     name: 'actions',
     register(pluginServer: Hapi.Server) {
-      pluginServer.expose('registerType', actionTypeService.registerType);
-      pluginServer.expose('listTypes', actionTypeService.listTypes);
+      pluginServer.expose('registerType', actionTypeRegistry.register);
+      pluginServer.expose('listTypes', actionTypeRegistry.listTypes);
```

Comment:
I think this should be ` pluginServer.expose('listTypes', actionTypeRegistry.list);`